_posts/2023-06-13-College-Contributor-Initiative.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title:  "College Contributor Initiative"
+authors:
+  - rednaksi
+date:   2023-06-13 09:00:00 -0700
+categories:
+  - announcements
+meta_keywords: Google Summer of Code, Major League Hacking, MIT Summer Camp, College Contributor Initiative, Coding Bootcamp
+
+meta_description: The OpenSearch Project launched the College Contributor Initiative (CCI) in March 2023, a program that provided students with opportunities to gain software development experience and contribute to open-source projects. The successful 12-week pilot, which concluded in May, led to a significant increase in external contributions, streamlined onboarding for developers, improved code quality, expanded project roadmaps, enhanced open-source maintenance experience, clarified project issues and increased community engagement, prompting an expansion of the program to new schools and locales.
```

Comment:
For meta - max is 3-4 targeted keywords/phrases and160 characters with spaces for descriptions. I will provide updated meta description and keywords to ensure they align with our strategy.

Reply:
@pajuric thank you! When do you think you will be able to provide the updated meta description?